roles/openshift_facts/library/openshift_facts.py

```diff
@@ -477,11 +477,7 @@ def set_selectors(facts):
             facts if they were not already present
 
     """
-    deployment_type = facts['common']['deployment_type']
-    if deployment_type == 'online':
-        selector = "type=infra"
-    else:
-        selector = "region=infra"
+    selector = "region=infra"
```

Comment:
This doesn't seem correct. `region` would relate to a region in the provider. `type` makes more sense and seems more correct since it references a openshift node type.

Reply:
I agree that the 'region' key for this variable is less than ideal.  However, this is what is already being used for everything but online.

The path of least resistance in this case (IMO) is to continue with region=infra here.

This selector value affects the following variables:
* openshift.hosted.router.selector
* openshift.hosted.registry.selector

These variables are used to determine which nodes to schedule work on.  The nodes are labeled with:
* openshift.node.labels

To maintain compatibility with existing installations, online could set the following:
```yaml
openshift_node_labels="{'type': 'infra'}"
openshift_hosted_registry_selector="type=infra"
openshift_hosted_registry_selector="type=infra"
```

Or, alternatively, update the labels on the nodes (it appears nodes can have more than one label, and these values don't seem to conflict, unless you're already labeling nodes with a region key)

Reply:
I believe online clusters use the selectors in the inventory like you mentioned above.